synapse/push/httppusher.py

```diff
@@ -97,6 +97,11 @@ def __init__(self, hs, pusherdict):
             pusherdict['pushkey'],
         )
 
+        if self.data is None:
+            raise PusherConfigException(
```

Comment:
will we actually log any less for a PusherConfigException than for a TypeError or whatever it was?

Reply:
Woops, I missed a code path where it wasn't handled.